aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -435,6 +435,15 @@
   "handlers": {
     "create": {
       "permissions": [
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeSubnets",
```

Comment:
I see in case of restore RDS DB instances using DB snapshots or point in time recovery.
RDS will need all ec2:* permissions?

ref: https://aws.amazon.com/premiumsupport/knowledge-center/rds-iam-least-privileges/

Reply:
I really would avoid using a wildcard in these permission specifications. Taking into account it was introduced to enforce the security, a wildcard throws us back in respect to this. To be frank, I'm not entirely happy we need to request more policies than 1:1 mapping to the API calls we're performing. Let's try to keep it explicit what is being requested here by enumerating all policies as far as we can keep it reasonable.